tests/ert_tests/ert3/evaluator/test_evaluator.py

```diff
@@ -39,9 +39,14 @@ def get_inputs(coeffs):
 
 @pytest.mark.requires_ert_storage
 @pytest.mark.parametrize("coeffs, expected", TEST_PARAMETRIZATION)
+@pytest.mark.parametrize(
+    "config, has_unix_config",
+    [["stages_config", True], ["function_stages_config", False]],
+)
```

Comment:
The test name indicates the difference between this test and the one removed. I think the name needs to be updated now...